src/components/Contributors.tsx

```diff
@@ -0,0 +1,30 @@
+/** @jsx jsx */
+import {jsx} from '@emotion/core'
+import React, {FunctionComponent} from 'react'
+
+type ContributorsProps = {
+  children: React.ReactNode
+  ariaLabel: string
+  title: string
+  buttonText: string
+  buttonStyles: string
+}
+const Contributors: FunctionComponent<ContributorsProps> = ({contributors}) => {
+  return (
+    <div className="flex content-start text-left justify-start lg:max-w-2xl mx-auto">
+      <h5 className="text-gray-500 uppercase tracking-widest text-extrabold justify-start m-0 mr-6">
+        Contributors
+      </h5>
+      <svg width="1.5rem" height="1.7rem">
+        <rect width="2px" height="100%" fill="#9E70C7" />
+      </svg>
+      {contributors.map((contributor, index) => (
+        <span key={index} className="justify-start m-0 mr-4">
```

Comment:
**suggestion:** using index as a key is not recommended. `${slugify(contributor)}-${index}`